src/phoenix/experimental/evals/models/litellm.py

```diff
@@ -0,0 +1,125 @@
+import logging
+from dataclasses import dataclass, field
+from typing import TYPE_CHECKING, Any, Dict, List, Optional
+
+from phoenix.experimental.evals.models.base import BaseEvalModel
+
+if TYPE_CHECKING:
+    from tiktoken import Encoding
+
+logger = logging.getLogger(__name__)
+
+
+@dataclass
+class LiteLLM(BaseEvalModel):
+    model_name: str = "gpt-3.5-turbo"
+    """The model name to use."""
+    temperature: float = 0.0
+    """What sampling temperature to use."""
+    max_tokens: int = 256
+    """The maximum number of tokens to generate in the completion."""
+    top_p: float = 1
+    """Total probability mass of tokens to consider at each step."""
+    num_retries: int = 6
+    """Maximum number to retry a model if an RateLimitError, OpenAIError, or
+    ServiceUnavailableError occurs."""
+    request_timeout: int = 60
+    """Maximum number of seconds to wait when retrying."""
+    extra_litellm_params: Dict[str, Any] = field(default_factory=dict)

```

Comment:
Might make sense to make this identical to the openai model: https://github.com/Arize-ai/phoenix/blob/814631042a04a98f9edd8964cb28317c426d144f/src/phoenix/experimental/evals/models/openai.py#L77